sympy/concrete/summations.py

```diff
@@ -253,6 +257,93 @@ def _eval_simplify(self, ratio, measure):
     def _eval_summation(self, f, x):
         return None
 
+    def is_convergent(self):
+        """
+        Convergence tests are used for checking the convergence of an
+        a series. There are various tests employed to check the
+        convergence. Like divergence test, root test, integral test,
+        alternating series test, comparison tests.
```

Comment:
Note that it returns `True` for convergent, `False` for divergent, and `None` for unknown.